attack/scripts/bashrc

```diff
@@ -122,17 +122,17 @@ starting_point() {
 
   #Determine if mode is node.
   if [[ "$MODE" == "node" ]]; then
-    NODE_TYPE="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | ."starting-point"."node-id"')"
+    NODE_TYPE="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | .startingPoint.nodeId')"
```

Comment:
do `jq` vars have to be prefixed with a `$`?  it would be neater if they weren't otherwise it's harder to distinguish between the `jq` variables and the `bash` variables

Reply:
Yes they do as per the `--arg` section [here](https://stedolan.github.io/jq/manual/).

Reply:
Yeah it's confusing but required as per the `--arg` section [here](https://stedolan.github.io/jq/manual/).

Reply:
Fair enough.  That was a stupid design decision on their part if you ask me! :(

Reply:
Yeah, `jq` doesn't follow a lot of BASH conventions (outputting `null` instead of nothing in `-r` mode for example) which is kind of annoying.